stable/awsebscsiprovisioner/values.yaml

```diff
@@ -49,13 +49,14 @@ provisioner:
   enableVolumeScheduling: false
   image:
     repository: "k8s.gcr.io/sig-storage/csi-provisioner"
-    tag: "v1.6.0"
-    tagK8sUpMinor17: v2.0.4
+    tag: "v1.6.1"
+    # same tag at the moment, but the logic should stay for now
```

Comment:
"but the logic should stay for now": not sure I understand what that mean in this context 🤔

Reply:
logic from within _helpers.tpl, so that we can as soon as nowing the fix for `v2.0.4` can easily roll up to that.